tests/common/models.py

```diff
@@ -58,18 +43,6 @@ def __init__(self, num_features: int = 1, num_classes: int = 2) -> None:
         self.fc2 = fc2
 
 
-@dataclasses.dataclass
-class SimpleModelHparams(ModelHparams):
```

Comment:
deleted objects here moved to `tests/hparams/common.py` or into the specific test if only used once to reduce fixture namespace area.